mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.js

```diff
@@ -28,13 +28,14 @@ const CheckboxGroup = forwardRef(
   (
     {
       id,
-      onBlur,
+      onBlur = () => {},
       onChange,
       options,
-      shouldAddOtherOptionWithClarification,
-      value,
-      SelectedMarkup,
-      shouldReturnEvent
+      optionsExcludedFromShowingSelectedMarkup = [],
+      shouldAddOtherOptionWithClarification = false,
+      value = [],
+      SelectedMarkup = undefined,
+      shouldReturnEvent = false
```

Comment:
react `defaultProps` wasnt working and apparently is slotted for depreciation, so I cant help but wonder if they arent bothering to fix its bugs. This fixed the bug you found.

Anyway, I'm leaning toward using the es6 way. Curious to hear the opinions of others in the web guild thread.

Reply:
interesting! also nice find. Will note not to use defaultProps from now on (or just use with caution). Going to read more into it